EIPS/eip-draft_security_considerations.md

```diff
@@ -14,27 +14,14 @@ requires: 1
 ## Simple Summary
 <!--"If you can't explain it simply, you don't understand it well enough." Provide a simplified and layman-accessible explanation of the EIP.-->
 
-This document describes an improvement to the Ethereum Improvment Proposal (EIP) system as defined in EIP-1 to explicitly embed security into the EIP process by visibly documenting security related information and considerations in a dedicated "Security Considerations" section of the EIP, adapted from the Request for Comments (RFC) system ([rfc7322 - section 4.8.5](https://tools.ietf.org/html/rfc7322#section-4.8.5)).
-
+Following the Constantinople fork postponement, all EIPs MUST include a "Security Considerations" section. Requirements include discussion of security considerations in the design, indicating which aspects of the protocol may be affected (networking, virtual machine, etc.), listing risks and how they have been mitigated. This proposal is adapted from the IETF's Request for Comments (RFC) system ([rfc7322 - section 4.8.5](https://tools.ietf.org/html/rfc7322#section-4.8.5)).
```

Comment:
Added brief reference to the motivations and specification to make it more of a summary.

Reply:
I think we need to rework this fragment: “Following the constantinople fork postponement…”. I was trying to keep the summary neutral to not unintentionally put blame on a subgroup for introducing an extra step in the EIP process as this might be contra productive. Keeping the opening neutral is paramount especially as the reason for explicitly adding security to the process is that is just was not yet manifested there. Even without the constantinople issues (actually a positive thing that it was found before) we should have had this process before.